From b04b00516fea37dddfa4d499e5672c203ff66cb1 Mon Sep 17 00:00:00 2001
From: Tom Stellard <thomas.stellard@amd.com>
Date: Thu, 15 May 2014 20:56:00 +0000
Subject: [PATCH 4/5] Merging r208908:

------------------------------------------------------------------------
r208908 | thomas.stellard | 2014-05-15 15:50:25 -0400 (Thu, 15 May 2014) | 7 lines

autoconf: Fix libLLVM-Major-Minor-Patch.so symlink

The symlink needs to point to a relative path, so we don't break
building in a chroot.

Tested-by: Laurent Carlier <lordheavym@gmail.org>

------------------------------------------------------------------------

git-svn-id: https://llvm.org/svn/llvm-project/llvm/branches/release_34@208917 91177308-0d34-0410-b5e6-96231b3b80d8
---
 Makefile.rules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.rules b/Makefile.rules
index 18882fa..fde77f9 100644
--- a/Makefile.rules
+++ b/Makefile.rules
@@ -1202,7 +1202,7 @@ $(DestSharedLib): $(LibName.SO) $(DestSharedLibDir)
 	$(Verb) $(INSTALL) $(LibName.SO) $(DestSharedLib)
 ifdef SHARED_ALIAS
 	$(Echo) Creating alias from $(DestSharedLib) to $(DestSharedAlias)
-	$(Verb) $(AliasTool) $(DestSharedLib) $(DestSharedAlias)
+	$(Verb) $(AliasTool) $(BaseLibName.SO) $(DestSharedAlias)
 endif
 
 uninstall-local::
-- 
1.9.3

